README.md

```diff
@@ -1,2 +1,45 @@
 # terraform-aws-sensor
+
 Terraform for Corelight's AWS Cloud Sensor Deployment.
+
+<img src="docs/overview.png" alt="overview">
+
+## Usage
+```terraform
+
+module "sensor" {
+  source = "github.com/corelight/terraform-aws-sensor"
+
+  auto_scaling_availability_zones = ["<first az>", "<second az>"]
+  aws_key_pair_name = "<key pair name>"
+  
+  # Request access to Corelight sensor AMI from you Account Executive
+  corelight_sensor_ami_id = "<sensor AMI ID>"
+  license_key = "<your Corelight sensor license key>"
+  management_subnet_id = "<management subnet>"
+  monitoring_subnet_id = "<monitoring subnet>"
+  sensor_api_password = "<password for the sensor api>"
+  vpc_id = "<vpc where the sensor auto scale group is deployed>"
+  
+  # (Optional) Fleet Subnet
+  fleet_subnet_id = "<subnet fleet is deployed in>"
```

Comment:
What is this used for? Not sure the sensor specifically needs to know about fleets subnet

Reply:
Looks like it controls security group rules, but shouldn't the yet to be created fleet terraform manage this?